docs/bdd_integration.rst

```diff
@@ -192,3 +193,73 @@ functions or check the :ref:`dataset <dataset>` module for more implementation d
 * :code:`[ENV:xxxx]`: Value of the OS environment variable xxxx
 * :code:`[FILE:xxxx]`: String with the content of the file in the path xxxx
 * :code:`[BASE64:xxxx]`: String with the base64 representation of the file content in the path xxxx
+
+In order to apply the string replacements in your code, import and call the corresponding function. E.g.::
+
+.. code:: console
+
+    from toolium.utils.dataset import map_param, replace_param
+
+    mapped_param = map_param('[TOOLIUM:Driver_chrome_driver_path]')
+    replaced_param = replace_param('[NOW - 1 MINUTES]')
+
+
+Type inference
+^^^^^^^^^^^^^^
+
+By default, the param replacements do not infer primitive types. To enable the type inference
+when using the `replace_param` function you have to add the following configuration:
+
+.. code:: console
+
+    [TestExecution]
+    infer_datatypes: true
```

Comment:
I have realized this is also not true in toolium, it should remain in toolium-telefonica. Here is quite the opposite, so we could say something like this:
```suggestion
Please note that, by default, the `replace_param` function tries to convert the resulting value to a native Python datatype. If that does not work for you, do not forget to set the `infer_param_type` parameter to `False`.
```